src/main/java/com/github/lmiljak/dowertefence/DowerTefence.java

```diff
@@ -8,8 +8,21 @@
  */
 public final class DowerTefence extends JavaPlugin {
 
+	/**
+	 * The default "parent" command of all commands associated with this plugin.
+	 * This means that every command starts with this String, followed by a sub
+	 * command. /[PARENT_COMMAND] [SUB_COMMAND] [arguments]. This command should
+	 * be equivalent to the one specified in the plugin.yml file.
+	 */
+	public static final String PARENT_COMMAND = "dt";
```

Comment:
perhaps we should also allow /dowertefence to be used, as some people prefer using the full plugin name over abbreviations